migration/v0.18.x-v0.19.x/upgrade-apps.md

```diff
@@ -105,3 +105,11 @@
    ```bash
    bin/ck8s ops kubectl sc delete po influxdb-0 -n influxdb-prometheus
    ```
+
+1. Remove grafana-ops chart with all the dashboards: `migration/v0.18.x-v0.19.x/remove-grafana-ops.sh`
```

Comment:
move this up above line 87, and remove lines 110-115

Reply:
Done!!